frontend/src/Admin/components/ShowJudgesTab.js

```diff
@@ -1,5 +1,90 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import { Row, Col, Progress, Button } from 'reactstrap'
+import { Link } from 'react-router-dom'
 
-export default function () {
-  return 'Judges'
+// Calculates a Bootstrap color for the given fractional progress, given
+// as a number from 0 to 1.
+const colorForProgress = progress => {
+  if (progress < 0.1) {
+    return 'danger'
+  }
+  if (progress < 0.25) {
+    return 'warning'
+  }
+  if (progress < 1) {
+    return null // default color
+  }
+  return 'success'
 }
+
+const ShowJudgesTab = ({ show }) => {
```

Comment:
2 Questions:

1. This doesn't seem to show the voting progress of Admins. Should it?

2. What currently happens on the backend to a Judge's votes if they are unassigned from a show? 
  a) Should the votes be kept or deleted?
  b) Should we disable Judge assignment after judging has started?

Reply:
1. I'm not sure... admins don't get 'assigned' to a show, so that's a bit tricky to instrument.
2. Currently the votes are kept, but I suspect that this shouldn't be the case since it will influence the aggregate score. Option 2 seems nice, but I feel like we should prefer freedom to assign judges last-minute over restriction

Reply:
1. I think it's fine at the moment to not list Admins. The main use case here is for Admins to see which Judges they need to bother to vote. I'm willing to push listing Admins off to "Future Work"

2. I agree with both your points. I think a happy medium would be to delete a Judge's votes if they are unassigned from a show. We could also show a confirmation dialog on the assignment page to make sure Admins know that.